webcompat/helpers.py

```diff
@@ -100,6 +100,18 @@ def get_user_info():
         session['avatar_url'] = gh_user.get('avatar_url')
 
 
+def version_string(json_section):
```

Comment:
`dict` is a built-in type/class, so we don't want to use that name by itself: https://docs.python.org/2/library/stdtypes.html#dict